src/Slider.jsx

```diff
@@ -35,110 +39,121 @@ class Button extends React.Component {
   disabled: PropTypes.bool,
+
   // a custom handle you can pass in
   handle: PropTypeReactComponent,
+
   // the maximum possible value
   max: PropTypes.number,
+
   // the minimum possible value
   min: PropTypes.number,
+
+  // step value
+  step: PropTypes.number,
+
   // called on click
   onClick: PropTypes.func,
+
   // called whenever the user is done changing values on the slider
   onChange: PropTypes.func,
+
   // called on key press
   onKeyPress: PropTypes.func,
+
   // called when you finish dragging a handle
   onSliderDragEnd: PropTypes.func,
+
   // called every time the slider is dragged and the value changes
   onSliderDragMove: PropTypes.func,
+
   // called when you start dragging a handle
   onSliderDragStart: PropTypes.func,
+
   // called whenever the user is actively changing the values on the slider
   // (dragging, clicked, keypress)
   onValuesUpdated: PropTypes.func,
+
   // the orientation
-  orientation: PropTypes.oneOf(['horizontal', 'vertical']),
+  orientation: OrientationPropType,
+
   // a component for rendering the pits
   pitComponent: PropTypeReactComponent,
+
   // the points that pits are rendered on
   pitPoints: PropTypeArrOfNumber,
+
   // a custom progress bar you can pass in
   progressBar: PropTypeReactComponent,
-  // should we snap?
-  snap: PropTypes.bool,
-  // the points we should snap to
-  snapPoints: PropTypeArrOfNumber,
-  // whether a proposed update is valid
-  getNextHandlePosition: PropTypes.func,
+
   // the values
   values: PropTypeArrOfNumber,
 };
 
+/* istanbul ignore next */
 const defaultProps = {
-  algorithm: linear,
+  autoAdjustVerticalPosition: true,
   className: '',
-  children: null,
+  algorithm: LinearScale,
   disabled: false,
-  handle: Button,
-  max: SliderConstants.PERCENT_FULL,
-  min: SliderConstants.PERCENT_EMPTY,
-  onClick: null,
-  onChange: null,
-  onKeyPress: null,
-  onSliderDragEnd: null,
-  onSliderDragMove: null,
-  onSliderDragStart: null,
-  onValuesUpdated: null,
-  orientation: 'horizontal',
-  pitComponent: null,
+  max: PERCENT_FULL,
+  min: PERCENT_EMPTY,
+  step: DEFAULT_STEP,
+  orientation: HORIZONTAL,
   pitPoints: [],
-  progressBar: 'div',
-  snap: false,
-  snapPoints: [],
-  getNextHandlePosition: null,
+  handle: DefaultHandle,
+  progressBar: DefaultProgressBar,
+
   values: [
-    SliderConstants.PERCENT_EMPTY,
+    PERCENT_EMPTY,
   ],
 };
 
-class Rheostat extends React.Component {
+/* istanbul ignore next */
+export class Rheostat extends React.Component {
   constructor(props) {
     super(props);
 
     const {
-      algorithm,
       max,
       min,
       values,
     } = this.props;
     this.state = {
-      className: getClassName(this.props),
-      handlePos: values.map(value => algorithm.getPosition(value, min, max)),
+      handlePos: values.map(value => this.props.algorithm.getPosition(value, min, max)),
       handleDimensions: 0,
-      // mousePos: null,
-      sliderBox: {},
+      mousePos: null,
```

Comment:
we determined that we could remove `mousePos`, right? Was the `sliderBox` delete intentional?